dojo/settings/settings.dist.py

```diff
@@ -119,11 +119,13 @@
     DD_SAML2_CREATE_USER=(bool, False),
     DD_SAML2_ATTRIBUTES_MAP=(dict, {
         # Change Email/UserName/FirstName/LastName to corresponding SAML2 userprofile attributes.
-        'Email': ('email', ),
-        'UserName': ('username', ),
-        'Firstname': ('first_name', ),
-        'Lastname': ('last_name', )
+        # format: SAML attrib:django_user_model
```

Comment:
@Maffooch I think you had two variables on the right side, will this solution work for you too? Or do you use local_settings.py or another way?

Reply:
I use local settings, so this can easily be overwritten there. I wonder if the env var mapping can be done with something like this to be consistent with the current  tuple mapping model.
```
DD_SAML2_ATTRIBUTES_MAP=(dict, {
         'Email': ['email'],
         'UserName': ['username'],
         'Firstname': ['first_name'],
         'Lastname': ['last_name']
 }),
```

Reply:
I assume it is the question of capabilities of https://django-environ.readthedocs.io/en/latest/, I was looking at the module, but not use is this the case